packages/influx-utilities/package.json

```
@@ -0,0 +1,65 @@
+{
+	"name": "@skyra/influx-utilities",
+	"version": "0.0.1",
+	"description": "A tiny opinionated abstraction layer for InfluxDB for interacting with a single organization",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
```

```suggestion
	"main": "dist/index.mjs",
```

Archid Components is primarily ESM only with the exception going to the few packages that have a more widespread userbase. Influx utilities will primarily be internal for us so I see no reason to not restrict it to ESM only.